p2p/service.py

```diff
@@ -15,8 +17,15 @@
 from p2p.exceptions import OperationCancelled
 
 
+if TYPE_CHECKING:
```

Comment:
Can we drop the `if TYPE_CHECKING` since this won't cause a circular import and we can just mark it with the `noqa: #401` to prevent linting errors?

Reply:
true that!